### ISSUE TITLE
Need nushell published as rpm and deb files for x86_64 (el8, el9, Debian and Ubuntu) platforms

### ISSUE THREAD

**Issue by emailstorbala** [nushell/nushell#12089]
### Related problem

In order to start using `nushell` officially in product as a replacement for `bash` shell, we need  `nushell` to be published in the popular distributions platforms like RHEL (el8/el9) and Debian (and Ubuntu).

### Describe the solution you'd like

Need `nushell` published for popular distributions (RHEL and Debian). Somewhere I saw an rpm for Fedora but missing for stable distros.

Eg:
el8 -> nushell-0.90.1-1.el8.x86_64.rpm
el9 -> nushell-0.90.1-1.el9.x86_64.rpm

![image](https://github.com/nushell/nushell/assets/12377900/e01c204a-9125-46b6-872b-856c95c65034)

**Comment by devyn** [comment]
I believe this is up to the individual distributions generally. If someone on the team wants to maintain an Ubuntu PPA that could be an option.

If you would like to install it on Ubuntu, you can download the static Linux binary from the releases page and either put it in `/usr/local/bin` or in your home directory somewhere. Or you can use `cargo install` to compile from source if you'd like, and that way it's easier to keep it up to date.

**Comment by emailstorbala** [comment]
> I believe this is up to the individual distributions generally. If someone on the team wants to maintain an Ubuntu PPA that could be an option.
> 
> If you would like to install it on Ubuntu, you can download the static Linux binary from the releases page and either put it in `/usr/local/bin` or in your home directory somewhere. Or you can use `cargo install` to compile from source if you'd like, and that way it's easier to keep it up to date.

@devyn If you see my description (screenshot), I have created the rpm (for el8. created under rockylinux 8.9) and the same is being used locally inside my organisation. But unfortunately, this way of preparing an upstream rpm locally and distributing (outside) as part of the product is highly discouraged and would not be allowed in many organisations. This is due to multiple reasons like authenticity of the rpm, security vulnerability tracking, etc. Hence I raised this request.

**Comment by emailstorbala** [comment]
@fdncred I am unable to understand why this story was closed with `completed` status. Was it intended to be marked as `will not be done` ?

**Comment by hustcer** [comment]
@emailstorbala 
>  I have created the rpm (for el8. created under rockylinux 8.9) and the same is being used locally inside my organisation.

BTW: We have [nushell/integrations](https://github.com/nushell/integrations) repo which was supposed to do such things. And the `src` branch will sync codes from Nushell `main` for each release. Maybe you can send a PR there?

**Comment by emailstorbala** [comment]
For anyone looking for nushell rpm/deb files for el8/el9/Debian(bookworm/trixie/Ubuntu), the same is available in https://github.com/emailstorbala/nushell-packages/releases/tag/v0.91.0

**Comment by hustcer** [comment]
rpm packages are available now, try to install it with:
```nu

echo "[gemfury-nushell]
name=Gemfury Nushell Repo
baseurl=https://yum.fury.io/nushell/
enabled=1
gpgcheck=0
gpgkey=https://yum.fury.io/nushell/gpg.key" | tee /etc/yum.repos.d/fury-nushell.repo
dnf install -y nushell
nu --version
```

Related issue: https://github.com/nushell/nushell/issues/15127
We welcome and appreciate your valuable feedback.

**Comment by atiq-cs** [comment]
> rpm packages are available now, try to install it with: fury-nushell.repo  

Is this up to date?  
On my system, fedora 42, it's still at 0.99.1 (I am using above instruction for fury's repo) while I see nushell's official latest is 0.106.0

**Comment by hustcer** [comment]
> > rpm packages are available now, try to install it with: fury-nushell.repo
> 
> Is this up to date? On my system, fedora 42, it's still at 0.99.1 (I am using above instruction for fury's repo) while I see nushell's official latest is 0.106.0

See the installation guide here: https://github.com/nushell/integrations

**Comment by hustcer** [comment]
> > rpm packages are available now, try to install it with: fury-nushell.repo
> 
> Is this up to date? On my system, fedora 42, it's still at 0.99.1 (I am using above instruction for fury's repo) while I see nushell's official latest is 0.106.0

Yes, it's up to date along with the official release. Look at the test workflow logs here: https://github.com/nushell/integrations/actions/runs/16562022303/job/46833462194#step:3:51

**Comment by atiq-cs** [comment]
Ah, I see the package name is "nushell" now. I was trying "nu". Thanks for looking up for me.
